### PR TITLE
Fix installation issues

### PR DESCRIPTION
**Pull request by mpougin**
fix installation issues in addition to #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ pip install -e .     # install python dependencies
 ### Running the app
 
 ```
+export PYTHONPATH="${PYTHONPATH}:/path/to/structure-property-visualizer"  # export pythonpath
```
Comment (ltalirz) [inline README.md:38]: Just a question: Is this really needed also when one runs this _from_ the folder of the repository?
I always thought the current working directory is added to the PYTHONPATH automatically

**Review by ltalirz — COMMENTED** [review]
great, thank you Miriam!

just one minor question

File: README.md (hunk 1 above)
Comment (mpougin) [reply README.md:38]: I was also surprised, but when I ran it inside the structure-property-visualizer folder, it complained that it couldn't import the modules:
`2022-09-14 22:16:21,702 Error running application handler <bokeh.application.handlers.directory.DirectoryHandler object at 0x7f8ff3dc2580>: No module named 'import_db'
File "main.py", line 15, in <module>:
from import_db import get_cif_content_from_disk as get_cif_str Traceback (most recent call last):
  File "/home/lsmo/miniconda/envs/test/lib/python3.9/site-packages/bokeh/application/handlers/code_runner.py", line 179, in run
    exec(self._code, module.__dict__)
  File "/home/lsmo/test/structure-property-visualizer/detail/main.py", line 15, in <module>
    from import_db import get_cif_content_from_disk as get_cif_str
ModuleNotFoundError: No module named 'import_db'`

Comment (mpougin) [reply README.md:38]: exporting the path fixed this for me and Elias

Comment (ltalirz) [reply README.md:38]: I see, thanks!

**Pull request merged**